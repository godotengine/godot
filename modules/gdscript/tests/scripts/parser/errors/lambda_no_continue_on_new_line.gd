# https://github.com/godotengine/godot/issues/73273

func not_called():
    var v
    v=func(): v=1
    in v
