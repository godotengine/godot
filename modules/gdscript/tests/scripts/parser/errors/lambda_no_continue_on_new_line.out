GDTEST_PARSER_ERROR
Expected statement, found "in" instead.
